pre-docs/bolt_running_plans.md

```diff
@@ -22,6 +22,7 @@ Note that, like `--nodes`, you can pass a comma-separated list of node names, wi
 
 **Tip**: Bolt is packaged with a collection of modules that contain useful plans to support common workflows. For details, see [Packaged modules](bolt_installing_modules.md#packaged-modules).
 
+**Note**: Check out some [plan examples](writing_yaml_plans.md#example-plans) to get an idea of some use cases for plans. 
```

Comment:
```suggestion
****Related information**

[Plan Examples](writing_yaml_plans.md#example-plans)
```

Reply:
Sorry to be nitpicky :( Example near the top of https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html

Reply:
Not a prob at all! Updated (decided to just amend the fix to my commit instead of committing your suggestion in case mellissa has any changes she wants to see).